src/OrleansTestingHost/TestClusterOptions.cs

```diff
@@ -17,14 +19,62 @@ namespace Orleans.TestingHost
     [Serializable]
     public class TestClusterOptions
     {
+        /// <summary>
+        /// Extended options to be used as fallbacks in the case that explicit options are not provided by the user.
+        /// </summary>
+        public class FallbackOptions
+        {
+            /// <summary>
+            /// Is set to true, by default the cluster will output traces in the console
```

Comment:
typo: Is -> If

Also, should we be using [the standard](http://stylecop.soyuz5.com/SA1623.html) "Gets or sets a value indicating whether the default cluster will output traces to the console."? I know it seems long-winded, but I wonder if we should decide on style standards for docs, layout, etc. If we do, we can mostly adhere to them automatically using ReSharper + StyleCop + some other plugins (AgentSmith, I think?)